bucket/netch.json

```diff
@@ -12,7 +12,7 @@
     "extract_dir": "Netch",
     "pre_install": [
         "ensure \"$dir\\data\" | Out-Null",
-        "if (!(Test-Path \"$persist_dir\\data\\settings.json\")) { Set-Content \"$dir\\data\\settings.json\" '{ \"CheckUpdateWhenOpened\": false }' -Encoding Ascii -Force }"
+        "if (!(Test-Path \"$persist_dir\\data\\settings.json\")) { Set-Content \"$dir\\data\\settings.json\" '{\"CheckUpdateWhenOpened\":false,\"Server\":[]}' -Encoding Ascii -Force }"
```

Comment:
```suggestion
        "if (!(Test-Path \"$persist_dir\\data\\settings.json\")) { Set-Content \"$dir\\data\\settings.json\" '{ \"CheckUpdateWhenOpened\": false, \"Server\": [] }' -Encoding Ascii -Force }"
```

Reply:
spaces make no sense, contents of `settings.json` will be overwritten when the app start.

Reply:
There is no reason to provide ugly minified json

Reply:
There is no reason to add meaningless spaces to a one-line JSON string which will be overwritten

Reply:
It is not about the final file, but about the maintainability of the manifest.